docs/using/changelog.md

```diff
@@ -0,0 +1,5 @@
+```{include} ../../CHANGELOG.md
```

Comment:
I want to keep the changelog file in the root of the repository, so I included it this way